react-validated-input-component/src/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Validator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isValid: false,
+      passLength: 0
+    };
+    this.passwordLength = this.passwordLength.bind(this);
+  }
+
+  passwordLength() {
+    this.setState({ passLength: this.state.passLength + 1 });
+    if (this.state.passLength === 7) {
+      this.render();
```

Comment:
in general, we never call our own `render()` method, we only let React do it

Reply:
you should also use this change handler (`passwordLength`) to store what the user has typed in state. Like in the `react-form-controls` exercise.